### PR TITLE
Blog post for deliverable on notifications

### PR DESCRIPTION
**Pull request by danidoni**

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit a17e95eec47c76ad80d47b0b2cd05bf75fe6fd7a

https://deploy-preview-269--openbuildservice.netlify.app

**Comment by danidoni** [comment]
> Good work! We should also write about the changes from [openSUSE/open-build-service#10220](https://github.com/openSUSE/open-build-service/pull/10220)

True!

File: _posts/development/2020-09-30-lots-of-notifications.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Lots of notifications?
```
[the remark this answers is not in the document]
Comment (danidoni) [reply _posts/development/2020-09-30-lots-of-notifications.md:3]: Fixing

**Comment by danidoni** [comment]
> Good work! We should also write about the changes from [openSUSE/open-build-service#10220](https://github.com/openSUSE/open-build-service/pull/10220)

I wrote some lines describing that change.

File: _posts/development/2020-09-30-lots-of-notifications.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Maybe you are part of some group, like `opensuse-review-team`, that receive lots of notifications. How do you deal with all those notifications on [your notifications page](https://build.opensuse.org/my/notifications)? Clicking one by one, on their &#10003; button?
+
+Nope!
+
+# Dealing With a Lot of Notifications
+
+Marking a lot of notifications as "read" one by one is a waste of time. And nobody likes wasting time!
+
+The notifications page now has some controls at the top of the list to perform actions, like marking as read, on several notifications at once.
```
Comment (krauselukas) [inline _posts/development/2020-09-30-lots-of-notifications.md:15]: Currently we only have one action :) maybe you can rephrase this a bit

File: _posts/development/2020-09-30-lots-of-notifications.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+The notifications list suffered from a bit of *information overload*: there was a lot to tell and it was scattered left and right. Now, the information is grouped together accordingly to the *what* and the *where*, so you just need a quick glance to know what's happening.
+
+<img src="/images/posts/2020-09-30/Redesign_of_notifications.png" alt="The 'where' is at the top, the 'what' is at the bottom" width="600px">
+
+# Wasting less time
+
+Talking again about wasting nobody's time, when clicking on a notification's subject, you could visit it but then you had to go back and mark it as "read". When you have lots of notifications this gets tiresome really fast.
+
+On that page, you now have a toolbar to mark the notification as "read" without having to go back to the notifications list page thus processing them faster.
```
Comment (krauselukas) [inline _posts/development/2020-09-30-lots-of-notifications.md:31]: On which page you have the toolbar? Maybe you can describe this a little more clear. So people know they find this on the view of the associated request etc.

File: _posts/development/2020-09-30-lots-of-notifications.md (hunk 2 above)
Comment (danidoni) [reply _posts/development/2020-09-30-lots-of-notifications.md:15]: Yeah, I will

File: _posts/development/2020-09-30-lots-of-notifications.md (hunk 3 above)
Comment (danidoni) [reply _posts/development/2020-09-30-lots-of-notifications.md]: Fixed

**Comment by danidoni** [comment]
@krauselukas I've addressed your suggestions. Let me know what do you think now.

File: _posts/development/2020-09-30-lots-of-notifications.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+The notifications list suffered from a bit of *information overload*: there was a lot to tell and it was scattered left and right. Now, the information is grouped together accordingly to the *what* and the *where*, so you just need a quick glance to know what's happening.
+
+<img src="/images/posts/2020-09-30/Redesign_of_notifications.png" alt="The 'where' is at the top, the 'what' is at the bottom" width="600px">
+
+# Wasting less time
+
+Talking again about wasting nobody's time, when clicking on a notification's subject, you could visit it but then you had to go back and mark it as "read". When you have lots of notifications this gets tiresome really fast.
+
+On the Request page, you now have a toolbar to mark the notification as "read" without having to go back to the notifications list, thus processing them faster.
```
Comment (krauselukas) [inline _posts/development/2020-09-30-lots-of-notifications.md:31]:
```suggestion
On the page the notification relates to, you now have a toolbar to mark the notification as "read" without having to go back to the notifications list, thus processing them faster.
```

Comment (krauselukas) [reply _posts/development/2020-09-30-lots-of-notifications.md:31]: Maybe more general, the toolbar is also available for comments

**Review by krauselukas — COMMENTED** [review]
Overall I like it, one suggestion would be to use the term 'waste time' maybe a little less :) if you use it to often I makes the blogpost sound a bit negative (and it is not)

File: _posts/development/2020-09-30-lots-of-notifications.md (hunk 4 above)
Comment (danidoni) [reply _posts/development/2020-09-30-lots-of-notifications.md:31]: True fact

**Comment by danidoni** [comment]
> Overall I like it, one suggestion would be to use the term 'waste time' maybe a little less :) if you use it to often I makes the blogpost sound a bit negative (and it is not)

I'm on it

> Overall I like it, one suggestion would be to use the term 'waste time' maybe a little less :) if you use it to often I makes the blogpost sound a bit negative (and it is not)

Addressed, how about now?

**Comment by hennevogel** [comment]
What about all the 'bugfix' cards?

**Comment by danidoni** [comment]
> What about all the 'bugfix' cards?

I thought about it. Most are small bugs and I felt that this was not the place to write a list of bugs fixes. The alternative was something in the line of "We also did several bug fixes" which per se didn't even need a section by itself, so I just omitted it.

File: _posts/development/2020-09-30-lots-of-notifications.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+The notifications list suffered from a bit of *information overload*: there was a lot to tell and it was scattered left
+and right. Now, the information is grouped together accordingly to the *what* and the *where*, so you just need a quick
+glance to know what's happening.
+
+<img src="/images/posts/2020-09-30/Redesign_of_notifications.png" alt="The 'where' is at the top, the 'what' is at the bottom" width="600px">
+
+# Getting things done
+
+When clicking on a notification's subject, you could visit it but then you had to go back and mark it as "read".
```
Comment (saraycp) [inline _posts/development/2020-09-30-lots-of-notifications.md:34]: I would rephrase this a bit, I don't think it is clear enough. Maybe a summarized version of this:

_By clicking on a notification's subject, you can visit the Request, Project or Package that caused the notification. After answering the comments or handling the Request, you could go back to the notifications list to mark them as "read"._

_From now on, on the same page, you have a toolbar to mark the notification as "read", without having to go 
back to the notifications list, thus processing them faster._

Comment (danidoni) [reply _posts/development/2020-09-30-lots-of-notifications.md:34]: I like it, stealing

Comment (danidoni) [reply _posts/development/2020-09-30-lots-of-notifications.md]: Review it again, please

**Comment by krauselukas** [comment]
@danidoni please squash the commit :) In my opinion its good to go :+1:

File: _posts/development/2020-09-30-lots-of-notifications.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+# Better Notifications
+
+The notifications list suffered from a bit of *information overload*: there was a lot to tell and it was scattered left
+and right. Now, the information is grouped together accordingly to the *what* and the *where*, so you just need a quick
+glance to know what's happening.
+
+<img src="/images/posts/2020-09-30/Redesign_of_notifications.png" alt="The 'where' is at the top, the 'what' is at the bottom" width="600px">
+
+# Getting things done
```
Comment (dmarcoux) [inline _posts/development/2020-09-30-lots-of-notifications.md:32]:
```suggestion
# Getting Things Done
```

File: _posts/development/2020-09-30-lots-of-notifications.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+By clicking on a notification's subject, you can visit the Request, Project or Package that caused that notification.
```
Comment (dmarcoux) [inline _posts/development/2020-09-30-lots-of-notifications.md:34]:
```suggestion
By clicking on a notification's subject, you can visit the request, project or package that caused that notification.
```

File: _posts/development/2020-09-30-lots-of-notifications.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+After answering the comments or handling the Request, you could go back to the notifications list to mark them as "read".
```
Comment (dmarcoux) [inline _posts/development/2020-09-30-lots-of-notifications.md:35]:
```suggestion
After answering the comments or handling the request, you could go back to the notifications list to mark them as "read".
```

File: _posts/development/2020-09-30-lots-of-notifications.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+From now on, on the same page, you have a toolbar to mark the notification as "read", without having to go
```
Comment (dmarcoux) [inline _posts/development/2020-09-30-lots-of-notifications.md:37]:
```suggestion
There's no need to do so since there is now a toolbar at the top of the page to mark the notification as "read".
```

Comment (dmarcoux) [inline _posts/development/2020-09-30-lots-of-notifications.md:37]:
```suggestion
There's no need to do so since there is now a toolbar at the top of the page to mark the notification as "read".
```

With this change, the line 38 should be removed.

Comment (danidoni) [reply _posts/development/2020-09-30-lots-of-notifications.md:37]: Okay

File: _posts/development/2020-09-30-lots-of-notifications.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+glance to know what's happening.
+
+<img src="/images/posts/2020-09-30/Redesign_of_notifications.png" alt="The 'where' is at the top, the 'what' is at the bottom" width="600px">
+
+# Getting Things Done
+
+By clicking on a notification's subject, you can visit the request, project or package that caused that notification.
+After answering the comments or handling the request, you could go back to the notifications list to mark them as "read".
+
+There's no need to do so since there is now a toolbar at the top of the page to mark the notification as "read".
```
Comment (dmarcoux) [inline _posts/development/2020-09-30-lots-of-notifications.md:37]: I would put this sentence in the same paragraph as the line 34.

File: _posts/development/2020-09-30-lots-of-notifications.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
+---
```
Comment (dmarcoux) [inline _posts/development/2020-09-30-lots-of-notifications.md:1]: The date of the post should be updated to today.

File: _posts/development/2020-09-30-lots-of-notifications.md (hunk 10 above)
Comment (danidoni) [reply _posts/development/2020-09-30-lots-of-notifications.md]: Done

File: _posts/development/2020-09-30-lots-of-notifications.md (hunk 11 above)
Comment (danidoni) [reply _posts/development/2020-09-30-lots-of-notifications.md]: Fixed

**Review by dmarcoux — APPROVED** [review]
Once the commits are squashed, let's ship it!

File: _posts/development/2020-10-05-lots-of-notifications.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+
+# Getting Things Done
+
+By clicking on a notification's subject, you can visit the request, project or package that caused that notification.
+After answering the comments or handling the request, you could go back to the notifications list to mark them as "read". 
+There's no need to do so since there is now a toolbar at the top of the page to do it.
+
+<img src="/images/posts/2020-10-05/Notifications_controls.png" width="600px">
+
+{% include partials/_how-to-join-the-beta-program.md %}
```
Comment (dmarcoux) [inline _posts/development/2020-10-05-lots-of-notifications.md:44]: I wouldn't include this here since it's already included in the blog post about the beta program linked in the first paragraph of this file.

File: _posts/development/2020-10-05-lots-of-notifications.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Lots of Notifications?
+category: development
+---
+
+This time we're gonna talk about [notifications](https://openbuildservice.org/2020/03/24/notifications-redesign/).
+Those of you that joined the [Beta program](https://openbuildservice.org/2018/10/04/the-beta-program/) gave us some
```
Comment (dmarcoux) [inline _posts/development/2020-10-05-lots-of-notifications.md:8]:
```suggestion
Those of you that joined the [beta program](https://openbuildservice.org/2018/10/04/the-beta-program/) gave us some 
```

File: _posts/development/2020-10-05-lots-of-notifications.md (hunk 12 above)
Comment (danidoni) [reply _posts/development/2020-10-05-lots-of-notifications.md:44]: Okay, I'll remove it

**Review by dmarcoux — APPROVED** [review]

**Review by saraycp — APPROVED** [review]
